jsonschema-maven-plugin/src/main/java/com/github/victools/jsonschema/plugin/maven/SchemaGeneratorMojo.java

```diff
@@ -522,9 +523,14 @@ private void addStandardModule(GeneratorModule module, SchemaGeneratorConfigBuil
             this.getLog().debug("- Adding Swagger 2.x Module");
             configBuilder.with(new Swagger2Module());
             break;
+        case "MicroProfileOpenApi3":
+            this.getLog().debug("- Adding MicroProfile OpenAPI 3.x Module");
+            configBuilder.with(new MicroProfileOpenApi3Module());
+            break;
         default:
             throw new MojoExecutionException("Error: Module does not have a name in "
-                    + "['Jackson', 'JakartaValidation', 'JavaxValidation', 'Swagger15', 'Swagger2'] or does not have a custom classname.");
+                    + "['Jackson', 'JakartaValidation', 'JavaxValidation', 'Swagger15', 'Swagger2', 'MicroProfileOpenApi3']"
```

Comment:
**suggestion**: I'm a big fan of a deterministic (and when in doubt: alphabetical) order. 😄
```suggestion
                    + "['Jackson', 'JakartaValidation', 'JavaxValidation', 'MicroProfileOpenApi3', 'Swagger15', 'Swagger2']"
```